lib/aws/rails/sqs_active_job/configuration.rb

```diff
@@ -158,6 +157,18 @@ def config_file_path(options)
         def user_agent
           "ft/aws-sdk-rails-activejob/#{Aws::Rails::VERSION}"
         end
+
+        def load_yaml(file_path)
+          require "erb"
+
+          # Avoid incompatible changes with Psych 4.0.0
+          # https://bugs.ruby-lang.org/issues/17866
+          if YAML.respond_to?(:unsafe_load)
+            YAML.unsafe_load(ERB.new(File.read(file_path)).result)
```

Comment:
As this code was originally take from rails, I'd prefer to take their fix for the issue and use `YAML.load(..., aliases: true)` as in https://github.com/rails/rails/pull/42249

Reply:
Agree... Let's not use unsafe loading!

Reply:
@alextwoods @mullermp 
Got it! I've changed code to use the Rails' approach you suggested. [19fafe7](https://github.com/aws/aws-sdk-rails/pull/91/commits/19fafe76993824b6c67598b00706f6938ec888a5)
Thank you for pointing it out.